ios/FluentUI/Controls/SearchBar.swift

```diff
@@ -14,6 +14,7 @@ public extension Colors {
             public static var cancelButton = UIColor(light: textSecondary, dark: LightContent.cancelButton)
             public static var clearIcon = UIColor(light: iconPrimary, dark: LightContent.clearIcon)
             public static var placeholderText = UIColor(light: textSecondary, dark: LightContent.placeholderText)
+            public static var progressSpinner: UIColor = DarkContent.searchIcon
```

Comment:
@Victa already mentioned the colors are incorrect:

Light mode / primary (= blue nav bar): Accent color Tint 10
Light mode / white: Gray 300
Dark mode / default: Gray 100

This is being used on the Light mode/ white, right? It should be gray300 instead of the current gray500.

DarkContent {
public static var **searchIcon** = UIColor(light: **iconPrimary**, dark: LightContent.searchIcon)
}

@objc public static let **iconPrimary** = UIColor(light: **gray500**, lightHighContrast: gray700, dark: .white)

Reply:
Also, could you update the screenshots once you fix the spinner colors? Thank you! :)

Reply:
updated colors for spinner.

---
In reply to: [468189125](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r468189125) [](ancestors = 468189125)

Reply:
Updated screenshots.

---
In reply to: [468196497](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r468196497) [](ancestors = 468196497)